static/js/operation.js

```diff
@@ -165,89 +127,69 @@ function parseAssembly(text, data) {
   var length = 0;
   var gc = 0;
   var coverage = 0;
-  var contig_length = 0;
-  var GC_count = 0;
+  var currentContigLength = 0;
+  var gcCount = 0;
 
   for (var i = 0; i < lines.length; i++) {
     var line = lines[i];
-    if (line.toUpperCase().match(/^[ATGCU]+$/g)) { // Checking if the current line is not a contig title
-      [GC_count, contig_length] = calculateGCAndLength(line, GC_count, contig_length);
-      gc = ((GC_count/contig_length).toFixed(2)).toString(); // idnetified last component of contig title
+    if (line.search(/^>{1}/gm) === 0) { // Checking if the current line is a contig title
+        [id, length, coverage] = parseContigTitles(line, format);
+        if (length <= minContigLength) 
```

Comment:
I think the threshold should be kept, therefore this comparison should be `<` instead of `<=`.